packages/angular/src/app/theme/theme.ts

```diff
@@ -0,0 +1,28 @@
+import { currentTheme as currentVizTheme, refreshTheme } from 'devextreme/viz/themes';
```

Comment:
It seems like a service. Create service and move this functional to the service

Reply:
+1 Angular blogs recommend the same technique: [theme switching](https://indepth.dev/posts/1218/lets-implement-a-theme-switch-like-the-angular-material-site#why-a-theme-switch)